packages/crayons-styles/package.json

```diff
@@ -18,6 +18,7 @@
   "main": "dist/index.js",
   "types": "dist/src/index.d.ts",
   "type": "module",
+  "files": ["dist", "css"],
```

Comment:
Hi, do we need `css` directory here ?. I suppose, we have a build script at the root of package.json `copy:themes` that copies the css folder from here and puts it inside `crayons-core/css`.

Reply:
Fixed.